kitty/config_data.py

```diff
@@ -256,6 +256,11 @@ def adjust_line_height(x):
 
 '''))
 
+o('show_ligatures_under_cursor', False, long_text=_('''
```

Comment:
This should be named disable_ligatures_under_cursor and default to False, to not change existing behavior.

Reply:
Originally I wanted to also have this option be inverted compared to `show_ligatures_under_cursor` but I couldn't think of a good short name. I thought of `hide_ligatures_under_cursor` but I found that to be confusing since it would suggest that nothing is visible at all. I like your suggestion `disable_ligatures_under_cursor` better. 👍